assets/src/hooks/useChannel.ts

```diff
@@ -143,3 +143,82 @@ export const useCheckedChannel = <T, U>({
   ])
   return state
 }
+
+export const useCheckedTwoWayChannel = <T, U>({
```

Comment:
**question:** do you think it would be possible / sensible to make this a special case of `useCheckedChannel`? It seems like a user of the hook could just ignore the `pushUpdate` part of the return value if they don't need to send any data.

Reply:
I think it is, I was hesitant to increase the scope of this PR but thinking about it more it should be a pretty minimal change.